notes/returned-mail.vbs

```diff
@@ -161,44 +161,55 @@ If snap_case = True Then snap_or_cash_case = True
-      EditBox 85, 75, 195, 15, returned_mail
-      EditBox 85, 95, 195, 15, verifications_requested
-      EditBox 85, 115, 195, 15, other_notes
-      EditBox 85, 135, 45, 15, due_date
-      EditBox 365, 115, 195, 15, notes_on_address
+    BeginDialog Dialog1, 0, 0, 566, 155, "RETURNED MAIL "
+      DropListBox 220, 15, 55, 15, "YES"+chr(9)+"NO", residential_address_confirmed
+      DropListBox 500, 15, 55, 15, "YES"+chr(9)+"NO", mailing_address_confirmed
+      EditBox 90, 75, 190, 15, returned_mail
       Text 10, 25, 200, 25, resi_addr_street_full
       Text 10, 35, 210, 15, resi_addr_city &  ", "  & resi_addr_state & " "   & resi_addr_zip
       Text 290, 25, 200, 25, mail_street_full
-      If mail_city <> "_______________" Then Text 290, 35, 200, 10, mail_city_line & ", "  & mail_state_line &  " "  & mail_zip_line
+      If trim(mail_city_line) <> "" THEN Text 290, 35, 200, 10, mail_city_line & ", "  & mail_state_line &  " "  & mail_zip_line
       GroupBox 285, 5, 275, 65, "Mailing Address"
       Text 290, 15, 210, 10, "Is this the address that the agency attempted to deliver mail to?"
       Text 10, 15, 210, 10, "Is this the address that the agency attempted to deliver mail to?"
-      Text 10, 55, 50, 10, "Future Date:"
+      IF addr_future_date <> "" THEN Text 10, 55, 50, 10, "Future Date:"
       Text 60, 55, 70, 10, addr_future_date
       Text 290, 55, 50, 10, "Effective Date:"
       Text 345, 55, 70, 10, addr_eff_date
       Text 5, 80, 65, 10, "What was returned:"
-      Text 290, 120, 65, 10, "Notes on address:"
-      GroupBox 5, 5, 275, 65, "Residential Address in Maxis"
+      GroupBox 5, 5, 275, 65, "Residential Address"
       GroupBox 285, 70, 275, 35, "Note:"
-      Text 290, 80, 265, 10, "Send out a Request for Contact(Verification Request Form - DHS 2919)"
-      Text 290, 90, 170, 10, "to the new address from the returned mail envelope"
+      Text 290, 80, 265, 10, "Send out a Request for Contact(Verification Request Form - DHS 2919) to the"
+      Text 290, 90, 170, 10, "correct address and ensure STAT/ADDR is updated."
       Text 5, 120, 40, 10, "Other notes:"
-      Text 5, 140, 35, 10, "Due date:"
-      Text 5, 100, 75, 10, "Verification requested:"
+      IF ADDR_actions = "address confirmed - received in error" THEN
+      	Text 5, 140, 200, 10, "How did you confirm the returned mail was received in error?"
+       	EditBox 220, 135, 215, 15, received_error_confirmation
+		Text 5, 100, 40, 10, "Other notes:"
+  		EditBox 90, 95, 190, 15, other_notes
+  		Text 5, 120, 65, 10, "Notes on address:"
+  		EditBox 90, 115, 190, 15, notes_on_address
+      ELSE
+      	Text 5, 100, 85, 10, "Verification(s) pending:"
+      	Text 5, 140, 35, 10, "Due date:"
+		Text 5, 120, 40, 10, "Other notes:"
+		Text 290, 120, 65, 10, "Notes on address:"
+      	EditBox 90, 95, 190, 15, verifications_requested
+      	EditBox 90, 135, 45, 15, due_date
+      	EditBox 90, 115, 190, 15, other_notes
```

Comment:
other_notes - update the tab order. It initially skips other_notes.

Reply:
this is in an else statement  can we review together?

Reply:
attempted to updated but it is in an else staetment